appveyor.yml

```diff
@@ -46,7 +46,7 @@ install:
 before_build: |
   if not exist build mkdir build
   cd build
-  cmake -G "Visual Studio 14 2015 Win64" .. -DALETH_INTERPRETER_SHARED=ON
```

Comment:
Ok, but this should stay :)

Reply:
Then testeth doesn't start because dll is required. Do you mean we should find the way to fix this?

Reply:
It should not be required, but maybe there is a build issue. The static build produces `aleth-interpreter.lib` and the DLL build produces `aleth-interpreter.lib` and `aleth-interpreter.dll`. Maybe the second build overwrites the static build. This should be handled separately.